### PR TITLE
Adding pyfftlog recipe

### PR DESCRIPTION
**Pull request by prisae**
Checklist
- [x] Title of this PR is meaningful
- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vendor other packages.
- [ ] ~If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by prisae** [comment]
I think this is ready to go @conda-forge/help-python. It is a very simple, one-file package where the only dependency is SciPy/NumPy.

File: recipes/pyfftlog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8b299e45be2e25c1facc2aa67b6895ec658e7083b098fe4fd9d81b92884d39b3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  preserve_egg_dir: true
```
Comment (nehaljwani) [inline recipes/pyfftlog/meta.yaml:16]: Why is this necessary?

Comment (prisae) [reply recipes/pyfftlog/meta.yaml:16]: I have it in my other ones, because of `conda-forge` and `setuptools_scm`, but I am not sure if it is required.

See
https://github.com/conda-forge/staged-recipes/pull/10087#discussion_r344346512

Comment (nehaljwani) [reply recipes/pyfftlog/meta.yaml:16]: I applied this diff:
```diff
diff --git a/recipes/pyfftlog/meta.yaml b/recipes/pyfftlog/meta.yaml
index d2d270b1a..4a89ec3b2 100644
--- a/recipes/pyfftlog/meta.yaml
+++ b/recipes/pyfftlog/meta.yaml
@@ -13,7 +13,7 @@ build:
   noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  preserve_egg_dir: true
+#  preserve_egg_dir: true

 requirements:
   host:
```
Then built the package and checked:
```bash
tar -xf /Users/nwani/.conda/envs/cf/conda-bld/noarch/pyfftlog-0.2.0-py_0.tar.bz2 site-packages/pyfftlog/version.py && cat site-packages/pyfftlog/version.py
# coding: utf-8
# file generated by setuptools_scm
# don't change, don't track in version control
version = '0.2.0'
```

So I believe it is not needed.

Comment (prisae) [reply recipes/pyfftlog/meta.yaml:16]: Thanks @nehaljwani . Worth a try and we'll find out.

**Comment by nehaljwani** [comment]
Thanks @prisae

**Comment by prisae** [comment]
Thank you @nehaljwani 

I just tested and it works fine. I will remove the `preserve_egg_dir: true` line in my other packages with the next new version.
